build-system/tasks/runtime-test.js

```diff
@@ -334,20 +326,29 @@ gulp.task('test', 'Runs tests',
     'grep': '  Runs tests that match the pattern',
     'files': '  Runs tests for specific files',
     'randomize': '  Runs entire test suite in random order',
-    'testlist': '  Runs tests specified in JSON by supplied file',
+    'seed': '  Seeds the test order randomization. Use with --randomize ' +
+        'or --a4a',
     'glob': '  Explicitly expands test paths using glob before passing ' +
         'to Karma',
     'nohelp': '  Silence help messages that are printed prior to test run',
+    'a4a': '  Runs all A4A tests',
   }
 });
 
 
-function shuffleArray(array) {
-    for (var i = array.length - 1; i > 0; i--) {
-        var j = Math.floor(Math.random() * (i + 1));
-        var temp = array[i];
-        array[i] = array[j];
-        array[j] = temp;
-    }
-    return array;
+function shuffleArray(array, seed) {
```

Comment:
Instead of implementing this, may I suggest you use `shuffle-seed` at https://www.npmjs.com/package/shuffle-seed ?

Reply:
oo neat, switched to that